subprojects/inline/inline.gradle

```diff
@@ -0,0 +1,18 @@
+description = "Mockito preconfigured inline mock maker"
```

Comment:
Are we sure we want this? For the moment, this is the best alternative. If we removed the artifact, we would probably rather announce it somewhere. It will probably not be before Mockito 3 anyways where people are naturally going through the release notes.